tracer/src/Datadog.Trace/AppSec/RcmModels/AsmData/Data.cs

```diff
@@ -0,0 +1,20 @@
+// <copyright file="Data.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
```

Comment:
Add `#nullable enable` to all new files.

Reply:
fixed in [36c59a6](https://github.com/DataDog/dd-trace-dotnet/pull/3087/commits/36c59a66e622abf5b4311fd6f255df824735b147)